Bias-Correction/bias_correction.ipynb

```diff
@@ -54,28 +35,41 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
-    "hist_file = '../../Seth Comparison/uas.hist.CanESM2.CRCM5-UQAM.day.NAM-44i.raw.nc'\n",
-    "ds_hist = xr.open_dataset(hist_file)"
+    "path = '/glade/u/home/jkent/data/bias_correction_CO'\n",
+    "\n",
+    "hist_file = path + '/uas.hist.CanESM2.CRCM5-UQAM.day.NAM-44i.raw.Colorado.nc'\n",
+    "rcp85_file = path + '/uas.rcp85.CanESM2.CRCM5-UQAM.day.NAM-44i.raw.Colorado.nc'\n",
+    "meas_file = path + '/uas.gridMET.NAM-44i.Colorado.nc'"
```

Comment:
@jukent,  #38 addressed this issue of needing to use different paths depending on whether you are on Cheyenne/Casper or not. 

As a result of #38, theses lines aren't needed. We should be able to add this notebook to the list of notebooks to tests: https://github.com/NCAR/notebook-gallery/blob/97bf3a32e1d55978d37c562747b56c3aa876c7fa/.circleci/config.yml#L35